src/HyperFormula.ts

```diff
@@ -1157,6 +1164,7 @@ export class HyperFormula implements TypedEmitter {
    *
    * @fires [[valuesUpdated]] if recalculation was triggered by this change
    *
+   * @throws [[NoSheetWithIdError]] when the given sheet ID does not exist
    * @throws [[InvalidArgumentsError]] when the value is not an array of arrays or a raw cell value
    * @throws [[SheetSizeLimitExceededError]] when performing this operation would result in sheet size limits exceeding
    * @throws [[ExpectedValueOfTypeError]] if topLeftCornerAddress argument is of wrong type
```

Comment:
probably out of scope, but I've noticed a line below has no error type

Reply:
@wojciechczerniak - good spot, added a card to investigate it: https://github.com/handsontable/hyperformula/projects/1#card-63826923